nucleus/dataset.py

```diff
@@ -738,6 +740,11 @@ def delete_item(self, reference_id: str) -> dict:
             requests.delete,
         )
 
+    @sanitize_string_args
+    def delete_scene(self, reference_id: str):
```

Comment:
Wondering if it makes more sense to just have a `delete_scenes(self, reference_ids: List[str])` method. We should also adapt the interface or at least add this method for the other data types to make it easier for the user and interate through the scenes internally rather then on the user side. Wdyt @gatli ?

Reply:
Can also do this in a second PR actually. So let's go ahead and merge once the endpoint is deployed.

Reply:
Agreed, that is probably a more natural interface. Let's add it to shortcut!